src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -12,8 +13,9 @@ import cromwell.binding.values.{WdlArray, WdlCallOutputsObject, WdlValue}
 import cromwell.engine.CallActor.CallActorMessage
 import cromwell.engine.ExecutionIndex._
 import cromwell.engine.ExecutionStatus.ExecutionStatus
+import cromwell.engine.Hashing._
 import cromwell.engine._
-import cromwell.engine.backend.{BackendCall, Backend, JobKey}
+import cromwell.engine.backend.{Backend, BackendCall, JobKey, LocalFileSystemBackendCall}
```

Comment:
I feel uncomfortable with WorkflowActor having a specific BackendCall type hardcoded in